### DISCUSSION TITLE
No relevant user found error when using default user + pass

### DISCUSSION THREAD

**Discussion by LawMixer** [NginxProxyManager/nginx-proxy-manager#2255] [Q&A]
I'm currently facing a problem where I'm trying to login to nginx proxy manager, and I use the default admin pass and user, and it's not working. I am local hosting it, and every single tutorial is for docker.

**Comment by mstarr0928** [comment]
Did you ever figure out an answer. I am encountering the same issue and I cant seem to figure it out.
